### PR TITLE
Fix some javadoc

### PR DESCRIPTION
**Pull request by matthias-ronge**
This commit fixes some javadoc comments.

### CODE REVIEW

File: Goobi/src/org/kitodo/production/lugh/ld/AccessibleObject.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 package org.kitodo.production.lugh.ld;
 
+import java.nio.BufferOverflowException;
```
Comment (stweil) [inline Goobi/src/org/kitodo/production/lugh/ld/AccessibleObject.java]: How is this modification related to the subject?

Comment (henning-gerhardt) [inline Goobi/src/org/kitodo/production/lugh/ld/AccessibleObject.java]: `BufferOverflowException` is used inside a [JavaDoc comment](https://github.com/matthias-ronge/kitodo-production/blob/483fefbf78c872002dedcf2cf8996731df13ce3e/Goobi/src/org/kitodo/production/lugh/ld/AccessibleObject.java#L30) and a good IDE mentioned the programmer that this symbol `BufferOverflowException` could not be resolved. Matthias try to fix that and as this it is related to pull request title.
